### ISSUE TITLE
Shoov buttons area

### ISSUE THREAD

**Issue by mldk-tech** [shoov/shoov#53]
@amitaibu, @solker -I'd love to see the buttons to download images, DIFF and the PR, over each image to not scroll up and down every time the page to click on these buttons

**Comment by amitaibu** [comment]
Yes, my idea is that the action buttons would be in a sticky block that
moves.
